openapi.yaml

```
@@ -132,6 +172,39 @@ components:
       properties:
         asyncapi:
           $ref: '#/components/schemas/AsyncAPIDocument'
+    ConvertRequest:
+      type: object
+      required:
+        - asyncapi
+      properties:
+        asyncapi:
+          $ref: '#/components/schemas/AsyncAPIDocument'
+        version:
+          $ref: '#/components/schemas/SpecVersions'
+        language:
+          type: string
+          description: Language to be converted to.
+          enum:
+            - 'yaml'
+            - 'json'
```

Please also add support for `yml` extension (another extension for YAML) :)

```suggestion
          enum:
            - 'yaml'
            - 'yml'
            - 'json'
```